### PR TITLE
Shoot setup

### PR DESCRIPTION
**Pull request by Diaphteiros**
Next to a refactoring of the setup scripts, this PR introduces the possibility to deploy the Gardener on a shoot cluster. While this might sound semi-relevant at first, it is a first step to make the setup independent of Kubify and to enhance it to be able to deploy the Gardener to any kubernetes cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This is the installation manual for a simple Gardener setup. It is part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
 
+We do recommend this simplified setup for demonstration purposes only. For productive workloads we do recommend that all components (Gardener/Seed/Shoot) run in their own IaaS accounts and that network policies are enabled and properly tested on the seed clusters.
+
```
Comment (marwinski) [inline README.md:6]: see https://github.com/gardener/documentation/pull/36

**Review by marwinski — CHANGES_REQUESTED** [review]
as discussed plus one documentation change request

File: README.md (hunk 1 above)
Comment (Diaphteiros) [reply README.md:6]: Added a sentence that a documentation for this is work in progress. Will be updated to reference the blog entry once it is released.

**Review by marwinski — APPROVED** [review]
too big but looks ok at a first glance. Will be tested once Gardener 0.14.0 is applied

**Pull request merged**